core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/jaas-realm/main/module.xml

```diff
@@ -48,5 +44,6 @@
         <module name="org.jboss.logmanager"/>
         <module name="org.wildfly.common"/>
         <module name="org.wildfly.security.elytron-private" export="true"/>
+        <module name="org.wildfly.extension.elytron"/>
```

Comment:
The most equivalent configuration would be by re-exporting `org.wildfly.extension.elytron` from this module. We would like to want to limit it to the `org.wildfly.extension.elytron` package only, so use the following:

```
<module name="org.wildfly.extension.elytron" export="true">
            <exports>
                <exclude path="org/wildfly/extension/elytron/*"/>
            </exports>
        </module>
```